### DISCUSSION TITLE
Theme branches

### DISCUSSION THREAD

**Discussion by ratulkhan74** [cadentcom/pico-wp-block-theme#9] [General]
demo2 and dev branch has theme  content

**Comment by stratofax** [comment]
This post is quoted from #7 

@ratulkhan74 says:

>I have used these branches for our pico theme development
>
> -   dev
>-   demo
>-   demo2
>
>for our new theme i want to start with a new branch and we can remove block theme branch we can remove the demo branch also

@stratofax replied:



OK, here's what I'll do:

- [ ] remove the `block theme` branch
- [ ] remove the `demo` branch

I'll leave these branches for now:

- `dev`
- `demo2`

Do you still want me to review the Pull Request (PR) on the `demo2` branch?

_This post was originally on discussion #7_

**Comment by stratofax** [comment]
@ratulkhan74 does the code on `demo2` work at all? In other words, can we use it as an actual WordPress theme?

If so, I'll review PR #3
